src/PaymentGateways/PayPalStandard/PayPalStandard.php

```diff
@@ -2,18 +2,52 @@
 
 namespace Give\PaymentGateways\PayPalStandard;
 
-use Give\PaymentGateways\PaymentGateway;
+use Give\Framework\Http\Response\Types\RedirectResponse;
+use Give\Framework\PaymentGateways\Commands\RedirectOffsite;
+use Give\Framework\PaymentGateways\PaymentGateway;
+use Give\Helpers\Call;
+use Give\PaymentGateways\DataTransferObjects\GatewayPaymentData;
+use Give\PaymentGateways\PayPalStandard\Actions\CreatePayPalStandardPaymentURL;
+use Give\PaymentGateways\PayPalStandard\Actions\RedirectOffsitePayment;
+use Give\PaymentGateways\PayPalStandard\Controllers\PayPalStandardWebhook;
+use Give\PaymentGateways\PayPalStandard\Views\PayPalStandardBillingFields;
+use Give_Payment;
 
-class PayPalStandard implements PaymentGateway
+class PayPalStandard extends PaymentGateway
 {
+    public $routeMethods = [
+        'handleIpnNotification'
+    ];
+
+    public $secureRouteMethods = [
+        'handleSuccessPaymentReturn',
+        'handleFailedPaymentReturn'
+    ];
+
     /**
      * @inheritDoc
      */
-    public function getId()
+    public function getLegacyFormFieldMarkup($formId, $args)
```

Comment:
The 'getLegacyFormFieldMarkup' base method expects a return value of `string|bool`. The legacy markup should be returned as a string.

Reply:
fixed: https://github.com/impress-org/givewp/compare/9333cfc25f8e...eeba557da48c